language/move-prover/tests/testsuite.rs

```diff
@@ -72,9 +96,94 @@ fn test_runner(path: &Path) -> datatest_stable::Result<()> {
             .into());
         }
     }
+
     Ok(())
 }
 
+fn cvc4blacklisted(path: &Path) -> bool {
```

Comment:
This could be done nicer if you like so. See the use of extract_test_directives in the `let baseline_valid = ...`. You could have `!extract_test_directive("no-cvc4")?` and then just have a comment `// no-cvc4` + explanations why in the source of the test. But I leave it to you, not sure how much time pressure there is for getting this done.

Reply:
Clark can't land this, I think, because his affiliation with Facebook ended.  So, I'm going to try to land it and we can fix this up in a future PR.

Reply:
The PR workflow did not pass. I tried to rerun it yesterday because the failure looked like a flake, but it again failed. Otherwise I would have already landed.